runtime/lua/vim/treesitter/dev.lua

```diff
@@ -43,10 +43,29 @@ local TSTreeView = {}
 ---@param injections table<string, TSP.Injection> Mapping of node ids to root nodes
 ---                  of injected language trees (see explanation above)
 ---@param tree TSP.Node[] Output table containing a list of tables each representing a node in the tree
-local function traverse(node, depth, lang, injections, tree)
+---@param include_root? boolean Include the root nodes in the tree
+local function traverse(node, depth, lang, injections, tree, include_root)
```

Comment:
Please keep `tree` as the last parameter. It acts as an “inout” parameter, its position at the end of the argument list is intentional.